src/Kreta/TaskManager/Tests/Spec/Kreta/TaskManager/Application/Organization/CreateOrganizationHandlerSpec.php

```diff
@@ -0,0 +1,54 @@
+<?php
+
+namespace Spec\Kreta\TaskManager\Application\Organization;
+
+use Kreta\TaskManager\Application\Organization\CreateOrganizationCommand;
+use Kreta\TaskManager\Application\Organization\CreateOrganizationHandler;
+use Kreta\TaskManager\Domain\Model\Organization\Organization;
+use Kreta\TaskManager\Domain\Model\Organization\OrganizationRepository;
+use Kreta\TaskManager\Domain\Model\Organization\Owner;
+use Kreta\TaskManager\Domain\Model\Organization\OwnerDoesNotExistException;
+use Kreta\TaskManager\Domain\Model\Organization\OwnerId;
+use Kreta\TaskManager\Domain\Model\Organization\OwnerRepository;
+use Kreta\TaskManager\Domain\Model\User\UserId;
+use PhpSpec\ObjectBehavior;
+use Prophecy\Argument;
+
+class CreateOrganizationHandlerSpec extends ObjectBehavior
```

Comment:
I am missing use cases when it is created without organization_id and organization_slug